package.json

```diff
@@ -25,7 +25,8 @@
   },
   "devDependencies": {
     "concurrently": "^4.0.1",
-    "cypress": "6.2.0",
+    "cypress": "7.7.0",
+    "cypress-iframe": "^1.0.1",
```

Comment:
```
Cypress.Commands.add("getIframeBody", locator => {
  // get the iframe > document > body
  // and retry until the body element is not empty
  cy.log("getIframeBody")

  return cy
    .get(locator, { log: false })
    .its("0.contentDocument.body", { log: false }).should("not.be.empty")
  // wraps "body" DOM element to allow
  // chaining more Cypress commands, like ".find(...)"
  // https://on.cypress.io/wrap
    .then(body => cy.wrap(body, { log: false }))
})
```

Reply:
please remove installed package and add this command to Cypress commands and use it as a base

Reply:
here is the example of usage
```
cy.getIframeBody("[data-target='account-settings--create-payment-method.cardNumberElement'] iframe")
    .find("input[name=cardnumber]")
    .type(card.number)
```